scripts/main/tabindex.js

```diff
@@ -2,92 +2,142 @@
+	// TODO @Hallenser: What did you intended by the TODO above? It seems as if the jQuery selector is already as short as possible?
+	const tmp = $(elem).find("[tabindex]");
 
 	// iterate over all elements and set tabindex to stored value (i.e. make is not focussable)
-	tmp.each(function (i, e) {
-		// TODO: shorter notation
-		a = $(e).data("tabindex-saved");
-		$(e).attr("tabindex", a);
-	});
+	tmp.each(
+		/**
+		 * @param {number} i - the index
+		 * @param {Element} e - the HTML element
+		 * @this {Element} - identical to `e`
+		 */
+		function (i, e) {
+			// TODO: shorter notation
+			// TODO @Hallenser: What do you intended by the TODO `short notation`? Moreover: Why do we use `this` and `e`? They refer to the identical instance of a HTML element.
+			const a = $(e).data("tabindex-saved");
+			$(e).attr("tabindex", a);
+		}
+	);
 };
 
+/**
+ * @param {jQuery} elem
+ * @param {boolean} [saveFocusElement=false]
+ * @returns {void}
+ */
 tabindex.makeUnfocusable = function (elem, saveFocusElement = false) {
 	if (!lychee.enable_tabindex) return;
 
-	// Todo: Make shorter noation
+	// Todo: Make shorter notation
 	// Get all elements which have a tabindex
-	let tmp = $(elem).find("[tabindex]");
+	const tmp = elem.find("[tabindex]");
 
 	// iterate over all elements and set tabindex to -1 (i.e. make is not focussable)
-	tmp.each(function (i, e) {
-		$(e).attr("tabindex", "-1");
-		// Save which element had focus before we make it unfocusable
-		if (saveFocusElement && $(e).is(":focus")) {
-			$(e).data("tabindex-focus", true);
-			// Remove focus
-			$(e).blur();
+	tmp.each(
+		/**
+		 * @param {number} i - the index
+		 * @param {Element} e - the HTML element
+		 */
+		function (i, e) {
+			$(e).attr("tabindex", "-1");
+			// Save which element had focus before we make it unfocusable
+			if (saveFocusElement && $(e).is(":focus")) {
+				$(e).data("tabindex-focus", true);
+				// Remove focus
+				$(e).blur();
+			}
 		}
-	});
+	);
 
 	// Disable input fields
-	$(elem).find("input").attr("disabled", "disabled");
+	elem.find("input").attr("disabled", "disabled");
 };
 
+/**
+ * @param {jQuery} elem
+ * @param {boolean} [restoreFocusElement=false]
+ * @returns {void}
+ */
 tabindex.makeFocusable = function (elem, restoreFocusElement = false) {
 	if (!lychee.enable_tabindex) return;
 
-	// Todo: Make shorter noation
+	// Todo: Make shorter notation
 	// Get all elements which have a tabindex
-	let tmp = $(elem).find("[data-tabindex]");
-
-	// iterate over all elements and set tabindex to stored value (i.e. make is not focussable)
-	tmp.each(function (i, e) {
-		$(e).attr("tabindex", $(e).data("tabindex"));
-		// restore focus elemente if wanted
-		if (restoreFocusElement) {
-			if ($(e).data("tabindex-focus") && lychee.active_focus_on_page_load) {
-				$(e).focus();
-				$(e).removeData("tabindex-focus");
+	const tmp = elem.find("[data-tabindex]");
+
+	// iterate over all elements and set tabindex to stored value (i.e. make is not focusable)
```

Comment:
```suggestion
	// iterate over all elements and set tabindex to stored value (i.e. make is focusable)
```

Reply:
Should that be "it" instead of "is"?

Reply:
I assume you are right. I haven't written this comment. But why do you want to delete the "not"? Anyway, the whole postfix in braces does not make sense to me. Settings the tabindex has nothing to do with the property of being focusable or not.

Reply:
Removed the confusing part in parentheses in: https://github.com/LycheeOrg/Lychee-front/pull/282/commits/620a74e9a15672b02687ac8f241a0a196df84648